crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1803,12 +1805,136 @@ impl std::fmt::Display for ProcError {
 
 impl std::error::Error for ProcError {}
 
+fn stackwalk_with_breakpad(
+    cfi_caches: procspawn::serde::Json<BTreeMap<DebugId, PathBuf>>,
```

Comment:
I would rather move the `procspawn` wrapper types out of this function signature into the closure that is specific to procspawn.

Reply:
Yeah, that seems like a good idea.